service-catalog/gcp-backup/README.md

```diff
@@ -0,0 +1,132 @@
+# GCP backup
+
+## Summary
+
+In GCP (Google Cloud Platform) we keep offsite backups for both Rust releases and crates
+to protect us against security threats that could involve losing crates or releases.
+These threats were identified in the [threat model].
+
+## Motivation
+
+While we have multiple measures in place to prevent accidental deletion of Rust releases or crates in AWS,
+e.g. bucket replication to a different region and restricted access, our current setup does not sufficiently protect us against a few threats:
+
+1. _AWS Account compromise_. The [threat model] for Rust's infrastructure, created by the Rust Foundation's security engineer, highlights the risk of an AWS account compromise.
```

Comment:
```suggestion
1. _AWS Account compromise_. The [threat model] highlights the risk of an AWS account compromise.
```

I suggest that we add the attribution above and then just keep referencing the threat model here.